xendit-java-library-example/src/main/java/ExampleCreateCustomer.java

```diff
@@ -0,0 +1,80 @@
+import com.xendit.Xendit;
+import com.xendit.exception.XenditException;
+import com.xendit.model.CustomerAddress;
+import com.xendit.model.Customer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ExampleCreateCustomer {
+  private static void createCustomer() {
+    try {
+      CustomerAddress customerAddress =  CustomerAddress.builder()
+          .country("ID")
+          .streetLine1("Jl. 123")
+          .streetLine2("Jl. 456")
+          .city("Jakarta Selatan")
+          .province("DKI Jakarta")
+          .state("-")
+          .postalCode("12345")
+          .category("None")
+          .isPreferred(true)
+          .build();
+      CustomerAddress[] customerAddressArray = new CustomerAddress[]{customerAddress};
+
+      String referenceId = "test-reference-id-2";
+      String email = "tes@tes.com";
+      String givenNames = "Given Names";
+      String nationality = "ID";
+      String dateOfBirth = "1995-12-30";
+
+      Customer customer = Customer.createCustomer(
+          referenceId,
+          null,
+          email,
+          givenNames,
+          null,
+          null,
+          null,
+          null,
+          nationality,
+          customerAddressArray,
+          dateOfBirth,
+          null
+      );
+      System.out.println(customer.getReferenceId());
+      System.out.println(customer.getEmail());
+      System.out.println(customer.getGivenNames());
+    } catch (XenditException e) {
+      e.printStackTrace();
+    }
+  }
+
+  private static void createCustomer2() {
```

Comment:
Can we have a more meaningful name for this example?

Reply:
got it